### PR TITLE
task 2 without using arrays

### PR DESCRIPTION
**Pull request by OlgaBelyakovich**
task completed

### CODE REVIEW

File: task1/task2.1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        static void Main(string[] args)
+        {
+            int min = 0, numI = 0, count=1, countforNums = 2, firstValue = 0, sum = 0;
+            double average;
+
+            Console.WriteLine("Enter the 1 number");
+            if (!int.TryParse(Console.ReadLine(), out firstValue))
+            {
+                Console.WriteLine("Enter correct number");
+                return;
```
Comment (dartenby) [inline task1/task2.1/Program.cs:20]: Allow user to keep entering

File: task1/task2.1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            }
+
+            do
+            {
+                Console.WriteLine($"Enter the {countforNums} number");
+                while (!int.TryParse(Console.ReadLine(), out numI))
+                {
+                    Console.WriteLine("Enter correct number");
+                }
+                count++;
```
Comment (dartenby) [inline task1/task2.1/Program.cs:30]: Do we need both? count and countforNums? please use one of them

File: task1/task2.1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                sum += numI;
+                if (min==0)
+                {
+                    min = firstValue;
+                }
+                if (min>numI)
+                {
+                    min = numI;
+                }
+                countforNums++;
```
Comment (dartenby) [inline task1/task2.1/Program.cs:40]: wrong naming - please read about camel case. right will  be - countForNums

File: task1/task2.1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            if (!int.TryParse(Console.ReadLine(), out firstValue))
+            {
+                Console.WriteLine("Enter correct number");
+                return;
+            }
+
+            do
+            {
+                Console.WriteLine($"Enter the {countforNums} number");
+                while (!int.TryParse(Console.ReadLine(), out numI))
```
Comment (dartenby) [inline task1/task2.1/Program.cs:26]: this structure can be reused below - please use methods for both

File: task1/task2.1/Program.cs [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                if (min==0)
+                {
+                    min = firstValue;
+                }
+                if (min>numI)
+                {
+                    min = numI;
+                }
+                countforNums++;
+                Console.WriteLine("Enter stop if you want to stop the programm, if you want to continue press smth else");
```
Comment (dartenby) [inline task1/task2.1/Program.cs:41]: it shows up only every second time - make it everytime

File: task1/task2.1/Program.cs [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+namespace task2._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int min = 0, numI = 0, count=1, countforNums = 2, firstValue = 0, sum = 0;
+            double average;
+
+            Console.WriteLine("Enter the 1 number");
```
Comment (dartenby) [inline task1/task2.1/Program.cs:16]: if I enter 5, 0, -1, 2 stop - then min value will be 2

File: task1/task2.1/Program.cs (hunk 1 above)
Comment (OlgaBelyakovich) [reply task1/task2.1/Program.cs]: fixed

File: task1/task2.1/Program.cs (hunk 2 above)
Comment (OlgaBelyakovich) [reply task1/task2.1/Program.cs]: fixed

File: task1/task2.1/Program.cs (hunk 3 above)
Comment (OlgaBelyakovich) [reply task1/task2.1/Program.cs]: fixed

**Pull request merged**